tests/views/test.product-pay.jsx

```diff
@@ -10,21 +12,33 @@ describe('ProductPay', function() {
   var fakePaymentProcessor;
   var defaultProductId = 'mozilla-concrete-brick';
   var braintreeToken = 'some-braintree-token';
+  var store;
   var userDefinedAmount = '10.00';
 
   beforeEach(function() {
     fakePaymentProcessor = sinon.spy();
+    store = createReduxStore();
   });
 
   function mountView({productId=defaultProductId} = {}) {
-    return TestUtils.renderIntoDocument(
-      <ProductPay
-        userDefinedAmount={userDefinedAmount}
-        braintreeToken={braintreeToken}
-        cardSubmissionErrors={{}}
-        productId={productId}
-        processPayment={fakePaymentProcessor}
-      />
+    var container = TestUtils.renderIntoDocument(
+      <Provider store={store}>
+        {() => {
+          return (
+            <ProductPay
+              userDefinedAmount={userDefinedAmount}
+              braintreeToken={braintreeToken}
+              cardSubmissionErrors={{}}
+              productId={productId}
+              processPayment={fakePaymentProcessor}
+            />
+          );
+        }}
+      </Provider>
+    );
+
+    return TestUtils.findRenderedComponentWithType(
```

Comment:
I was thinking this Provider/find pattern could probably be wrapped up a little better too